server_environment/__init__.py

```diff
@@ -0,0 +1,27 @@
+##############################################################################
```

Comment:
I think we can finally cleanup this headers too...

Reply:
@simahawk Okay, I can do that too. What license is it btw? The main module and manifest say GPL3+, but other headers here and there say AGPL.

Reply:
interesting.... @guewen do you have a more clear idea?

Reply:
It seems the old files and the manifest are all stating "GPL-3 or any later version" and only my new mixin (and their tests) is set as AGPL-3, the inconsistency is my fault then. We can change the headers with AGPL3 to match the rest if this is the easy way.